example/example-node.js

```diff
@@ -1,252 +1,681 @@
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                    <td style="width: 117pt; border-top: none; border-bottom: none; border-left: none; border-image: initial; border-right: 1pt solid windowtext; padding: 0in 5.4pt; height: 19.5px;"
+                        valign="top">
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                    <td style="width: 117pt; border-top: none; border-left: none; border-bottom: 1pt solid rgb(0, 176, 80); border-right: 1pt solid black; padding: 0in 5.4pt; height: 19.5px;"
+                        valign="top">
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                    <td style="width: 117pt; border-top: none; border-left: none; border-bottom: 1pt solid rgb(237, 125, 49); border-right: 1pt solid black; padding: 0in 5.4pt; height: 19.5px;"
+                        valign="top">
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                </tr>
+                <tr style="height: 20px;">
+                    <td style="width: 117pt; border: none; padding: 0in 5.4pt; height: 20px;" valign="top">
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                    <td style="width: 117pt; border-top: none; border-bottom: none; border-left: none; border-image: initial; border-right: 1pt solid windowtext; padding: 0in 5.4pt; height: 20px;"
+                        valign="top">
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                    <td style="width: 117pt; border-top: none; border-left: none; border-bottom: 1pt solid rgb(0, 176, 80); border-right: 1pt solid rgb(237, 125, 49); padding: 0in 5.4pt; height: 20px;"
+                        valign="top">
+                        <p style="margin: 0in; line-height: normal; font-size: 11pt; font-family: Calibri, sans-serif;">&nbsp;
+                        </p>
+                    </td>
+                    <td style="width: 117pt; border-top: none; border-left: none; border-bottom: 1pt solid rgb(237, 125, 49); border-right: 1pt solid rgb(237, 125, 49); padding: 0in 5.4pt; height: 20px;"
+                        valign="top">

```

Comment:
valign and background won't work as of this PR. This will be taken care of in the next PR. Same with padding and height (please make tickets in this repo for padding and height)